pkg/storage/sqlstorage/store_test.go

```diff
@@ -36,46 +36,17 @@ func TestStore(t *testing.T) {
 	}
 
 	for _, tf := range []testingFunction{
-		{
-			name: "SaveTransactions",
-			fn:   testSaveTransaction,
-		},
-		{
-			name: "DuplicatedTransaction",
-			fn:   testDuplicatedTransaction,
-		},
-		{
-			name: "LastLog",
-			fn:   testLastLog,
-		},
-		{
-			name: "CountAccounts",
-			fn:   testCountAccounts,
-		},
-		{
-			name: "GetAssetsVolumes",
-			fn:   testAggregateVolumes,
-		},
-		{
-			name: "GetAccounts",
-			fn:   testGetAccounts,
-		},
-		{
-			name: "TransactionsQuery",
-			fn:   testTransactions,
-		},
-		{
-			name: "GetTransaction",
-			fn:   testGetTransaction,
-		},
-		{
-			name: "Mapping",
-			fn:   testMapping,
-		},
-		{
-			name: "TooManyClient",
-			fn:   testTooManyClient,
-		},
+		{name: "AppendLog", fn: testAppendLog},
```

Comment:
Why this modification?

Reply:
that's from the tests @antoinegelloz added to the store logic i think

Reply:
@gfyrag @jdupas22 This new test `testAppendLog` is a combinaison of `testSaveTransaction` et `testDuplicatedTransaction` in one, because what is really tested in those two functions is the AppendLog Store method.

Reply:
@antoinegelloz My question was about inlining the struct fields ;)

Reply:
Oh ok ! I just find it nicer to have a view of the entire list of tests fitting in my screen.
Don't you think?